build/images/ovs/Dockerfile.ubi

```diff
@@ -0,0 +1,43 @@
+# OVS build scripts are only applicable for RHEL 7.x:
+# https://docs.openvswitch.org/en/latest/intro/install/fedora
+FROM centos:centos7 as ovs-rpms
+
+# Some patches may not apply cleanly if another version is provided.
+ARG OVS_VERSION=2.14.0
+
+# Install RPM tools and generic build dependencies.
+RUN yum update -y && yum install wget git yum-utils python38 rpm-build epel-release -y
+
+COPY apply-patches.sh /
+
+# Download OVS source code
+RUN wget -q -O - https://www.openvswitch.org/releases/openvswitch-$OVS_VERSION.tar.gz  | tar xz -C /tmp
+RUN cd /tmp/openvswitch* && \
+    /apply-patches.sh && \
+    sed -e "s/@VERSION@/$OVS_VERSION/" rhel/openvswitch-fedora.spec.in > /tmp/ovs.spec && \
+    yum-builddep -y /tmp/ovs.spec && ./boot.sh && \
+    ./configure --prefix=/usr --localstatedir=/var --sysconfdir=/etc && \
+    make rpm-fedora && mkdir -p /tmp/ovs-rpms && \
+    mv /tmp/openvswitch-$OVS_VERSION/rpm/rpmbuild/RPMS/*/*.rpm  /tmp/ovs-rpms && \
+    rm -rf /tmp/openvswitch*
```

Comment:
This is not strictly related to this PR, but if we have to handle multiple dockerfiles we should have a mechanism to generate them, as remembering to make changes in each dockerfile is error prone

Reply:
Good point. We need the make optimization for it later.